bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/service/BahmniDrugOrderService.java

```diff
@@ -38,4 +39,13 @@ List<DrugOrder> getInactiveDrugOrders(String patientUuid, Set<Concept> concepts,
 
     List<BahmniDrugOrder> getDrugOrders(String patientUuid, Boolean isActive, Set<Concept> conceptsToFilter, Set<Concept> conceptsToExclude,
                                         String patientProgramUuid) throws ParseException;
+
+    List<BahmniDrugOrder> getBahmniDrugOrdersForVisit(String patientUuid, String visitUuid);
+
+    Map<BahmniDrugOrder, String> getMergedDrugOrderMap(List<BahmniDrugOrder> drugOrderList);
```

Comment:
this method can be internally used from the getPrescriptionAsString() .. no need to declare and expose in interface/classes. can be private

Reply:
this method implementation does not need to be in BahmniDrugOrderServiceImpl, rather all the implementation can be in SharePrescriptionServiceImpl

Reply:
moved the method to SharePrescriptionServiceImpl and made as private